AI-and-Analytics/Features-and-Functionality/IntelTensorFlow_ModelZoo_Inference_with_FP32_Int8/README.md

```diff
@@ -1,122 +1,165 @@
+conda activate tensorflow
 ```
 
-## How to Build and Run
-1. Go to the code example location.<br>
-2. If you have GUI support, enter the command `jupyter notebook`. <br>
-or<br>
-a. If you do not have GUI support, open a remote shell and enter command `jupyter notebook --no-browser --port=8888`.<br>
-b. Open the command prompt where you have GUI support, and forward the port from host to client.<br>
-c. Enter `ssh -N -f -L localhost:8888:localhost:8888 <userid@hostname>`<br>
-d. Copy-paste the URL address from the host into your local browser to open the jupyter console.<br>
-3. Go to `ResNet50_Inference.ipynb` and run each cell to create synthetic data and run int8 inference.
-
----
-**NOTE**
+#### Activate Conda without Root Access (Optional)
 
-In the jupyter page, be sure to select the correct kernel. In this example, select 'Kernel' -> 'Change kernel' -> Python [conda env:tensorflow].
+You can choose to activate Conda environment without root access. To bypass root access to manage your Conda environment, clone and activate your desired Conda environment using the following commands similar to the following.
 
----
+```
+conda create --name user_tensorflow --clone tensorflow
+conda activate user_tensorflow
+```
 
-### **Request a Compute Node**
-In order to run on the DevCloud, you need to request a compute node using node properties such as: `gpu`, `xeon`, `fpga_compile`, `fpga_runtime` and others. For more information about the node properties, execute the `pbsnodes` command.
- This node information must be provided when submitting a job to run your sample in batch mode using the qsub command. When you see the qsub command in the Run section of the [Hello World instructions](https://devcloud.intel.com/oneapi/get_started/aiAnalyticsToolkitSamples/), change the command to fit the node you are using. Nodes which are in bold indicate they are compatible with this sample:
+#### Navigate to Model Zoo
 
-<!---Mark each compatible Node in BOLD-->
-| Node              | Command                                                 |
-| ----------------- | ------------------------------------------------------- |
-| GPU               | qsub -l nodes=1:gpu:ppn=2 -d . hello-world.sh           |
-| CPU               | qsub -l nodes=1:xeon:ppn=2 -d . hello-world.sh          |
-| FPGA Compile Time | qsub -l nodes=1:fpga\_compile:ppn=2 -d . hello-world.sh |
-| FPGA Runtime      | qsub -l nodes=1:fpga\_runtime:ppn=2 -d . hello-world.sh |
+Navigate to the Model Zoo for Intel® Architecture source directory. By default, it is in your installation path, like `/opt/intel/oneapi/modelzoo`. 
 
+1. View the available Model Zoo release versions for the AI Kit:
+   ```
+   ls /opt/intel/oneapi/modelzoo
+   1.8.0  latest
+   ```
+2. Navigate to the [Model Zoo Scripts](https://github.com/IntelAI/models/tree/master/benchmarks) GitHub repo to determine the preferred released version to run inference for ResNet50 or another supported topology.
```

Comment:
I'm not sure what the point of this step is. If the user clicks on this link in github, it will take them to the master branch of the online version of model zoo, which is almost always ahead of what is in AI Kit.  TBH, the `latest` step below is the best option unless we want to update this page with every AI Kit release like so:
```suggestion
2. Navigate to the [Model Zoo Scripts](https://github.com/IntelAI/models/tree/v2.11.0/benchmarks) GitHub repo to determine the preferred released version to run inference for ResNet50 or another supported topology.
```

The link above is not live, but it is the version that we are dropping into AI Kit 2023.1.1.